src/components /DummyGraph.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import moment from "moment";
+import { Chart } from "react-chartjs-2";
+import "chartjs-adapter-moment";
+import {
+  Chart as ChartJS,
+  LinearScale,
+  CategoryScale,
+  BarElement,
+  PointElement,
+  LineElement,
+  Legend,
+  Tooltip,
+  LineController,
+  BarController,
+  TimeScale,
+} from "chart.js";
+
+import uuid from "react-uuid";
+
+ChartJS.register(
+  LinearScale,
+  CategoryScale,
+  BarElement,
+  PointElement,
+  LineElement,
+  Legend,
+  Tooltip,
+  LineController,
+  BarController,
+  TimeScale,
+);
+
+const DummyGraph = () => {
+  const dataGraph = {
```

Comment:
static objects like so, always to be defined outside of components. Otherwise they get redefined after every rerender